ui/app/utils/openapi-helpers.ts

```diff
@@ -132,3 +132,37 @@ export function filterPathsByItemType(pathInfo: PathsInfo, itemType: string): Pa
     return itemType === path.itemType;
   });
 }
+
+/**
+ * This object maps model names to the openAPI path that hydrates the model, given the backend path.
+ */
+const OPENAPI_POWERED_MODELS = {
+  'role-ssh': (backend: string) => `/v1/${backend}/roles/example?help=1`,
+  'auth-config/azure': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/cert': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/gcp': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/github': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/jwt': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/kubernetes': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/ldap': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/okta': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'auth-config/radius': (backend: string) => `/v1/auth/${backend}/config?help=1`,
+  'kmip/config': (backend: string) => `/v1/${backend}/config?help=1`,
+  'kmip/role': (backend: string) => `/v1/${backend}/scope/example/role/example?help=1`,
+  'pki/role': (backend: string) => `/v1/${backend}/roles/example?help=1`,
+  'pki/tidy': (backend: string) => `/v1/${backend}/config/auto-tidy?help=1`,
+  'pki/sign-intermediate': (backend: string) => `/v1/${backend}/issuer/example/sign-intermediate?help=1`,
+  'pki/certificate/generate': (backend: string) => `/v1/${backend}/issue/example?help=1`,
+  'pki/certificate/sign': (backend: string) => `/v1/${backend}/sign/example?help=1`,
+  'pki/config/acme': (backend: string) => `/v1/${backend}/config/acme?help=1`,
+  'pki/config/cluster': (backend: string) => `/v1/${backend}/config/cluster?help=1`,
+  'pki/config/urls': (backend: string) => `/v1/${backend}/config/urls?help=1`,
+};
+
+export function getHelpUrlForModel(modelType: string, backend: string) {
+  const urlFn = OPENAPI_POWERED_MODELS[modelType as keyof typeof OPENAPI_POWERED_MODELS] as (
```

Comment:
do we need both keyof and typeof here?

Reply:
Yup, because `OPENAPI_POWERED_MODELS` is a const not an interface, so we need typeof so typescript is happy